recipes/minizip-ng/all/conanfile.py

```diff
@@ -0,0 +1,143 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class MinizipNgConan(ConanFile):
+    name = "minizip-ng"
+    description = "Fork of the popular zip manipulation library found in the zlib distribution."
+    topics = ("conan", "compression", "zip")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zlib-ng/minizip-ng"
+    license = "Zlib"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package", "pkg_config"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "mz_compatibility": [True, False],
+        "with_zlib": [True, False],
+        "with_bzip2": [True, False],
+        "with_lzma": [True, False],
+        "with_zstd": [True, False],
+        "with_openssl": [True, False],
+        "with_iconv": [True, False],
+        "with_libbsd": [True, False],
+        "with_libcomp": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "mz_compatibility": False,
+        "with_zlib": True,
+        "with_bzip2": True,
+        "with_lzma": True,
+        "with_zstd": True,
+        "with_openssl": True,
+        "with_iconv": True,
+        "with_libbsd": True,
+        "with_libcomp": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            del self.options.with_iconv
+            del self.options.with_libbsd
+
+    def build_requirements(self):
+        self.build_requires('pkgconf/1.7.3')
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+        if tools.is_apple_os(self.settings.os) and self.options.with_libcomp:
+            self.options.with_zlib = False
```

Comment:
```suggestion
        if tools.is_apple_os(self.settings.os) and self.options.with_libcomp:
            del self.options.with_zlib
```

If there is no option of zlib, why generate a package ID?